UPGRADE.md

```diff
@@ -109,9 +116,7 @@ if you are using docker infrastructure you can follow steps written below.
 
     `docker exec -it shopsys-framework-postgres pg_dumpall > backupfile`
 
-1. copy new version of `docker-compose.`:
-
-    `cp docker/conf/docker-compose.yml.dist -f docker-compose.yml`
+1. apply changes in `docker-compose.yml`, you can find them in a new version of [`docker-compose.yml.dist`](https://github.com/shopsys/shopsys/blob/v7.0.0-alpha5/docker/conf/docker-compose.yml.dist)
```

Comment:
In my opinion, it would be so much better if the link lead to the page where the actual changes are highlighted - I was able to achieve that by using the "compare" github feature but my link (`https://github.com/shopsys/shopsys/compare/v7.0.0-alpha4...v7.0.0-alpha5#diff-3f6467a573e3338ce2d634e10eea03df`) does not seem to be working properly so I am confused. What do you think? Also, it should somehow mention, that you need the apply the changes from the corresponding docker-compose.yml.dist

Reply:
I had the same idea, and I ran into the same issue - the link doesn't work properly. So I dediced to leave here just a link to new version. You can always copy it and see changes manually.

I added note for `docker-compose.yml.dist`